ci/pipelines/cf-for-k8s-contributions.yml

```diff
@@ -775,6 +776,137 @@ jobs:
         file: cf-for-k8s-ci/ci/tasks/delete-rds/task.yml
         params:
           PGPASSWORD: ((ci_k8s_aws_rds_database_password))
+- name: validate-pr-on-gke-external-blobstore
```

Comment:
There appear to be some indentation issues here that don't exist in the corresponding template. It might be worth commenting out the `fly set-pipeline` command from the `ci/configure` script and just running that to re-generate it?

Reply:
We re-run the `configure` script but did not see any differences in the generated pipeline.

Reply:
Interesting. I am concerned about the indentation for lines 816 - 909. The put to the `ready-pool` resource (lines 907 - 909) looks especially strange. We'll go ahead and merge this and will address any issues that show up when we set the pipeline.